docs/blog/2021-04-05-great-design-is-transparent.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Great Design is Transparent
+authors:
+  - name: Derek Schaller
+    url: https://github.com/dschaller
+    avatar: https://avatars1.githubusercontent.com/u/1004789?s=460&u=24deb32096e9f892cc91a6ff1ca1af50193b1fbd&v=4
+description: How we designed and then redesigned the Frontend of Clutch.
+image: https://user-images.githubusercontent.com/1004789/113597092-0eec7800-95f0-11eb-8f94-b953dd790c23.png
+hide_table_of_contents: false
+---
+
+Design is one of those things that is tricky but imperative to get right. While this may seem obvious, creating and implementing a great design is much harder than you might think.
+
+<!--truncate-->
+
+Why is design hard? Design is something that you usually start with when building a project. It is akin to blueprints of a house, determining not only what you will end up building but also the finishing details. This alone is not a problem; design is highly opinionated and there is no way to concretely say that a particular design is “correct”, especially so early on in a project. However, there is a group who can determine what the “right” design is for your product and they are your audience. Ideally you would provide various designs for them to interact with, collect feedback, and choose the best of the options but again this is difficult to do early on.
+
+Without being able to experiment and iterate, most of the design decisions are largely guesses as to what would work best for your audience and these will likely get your product to a place where the design is “good”. This isn’t bad, a good design is something to be proud of. Users will engage with your product and recognize the thought put into the various pieces and interactions. However, an even better experience to strive for would be a design that is transparent. This might seem counterintuitive; you might think that you want users to see and appreciate all of the work put into design when, in fact, you actually want the opposite. 
+
+A transparent design is one that results in a  user experience so intuitive and seamless that little thought is given to using the product, as there is no friction, and users are ultimately drawn back in the future. However, it’s imperative to note that it’s easy to forget that you don’t know what you don’t know, spending too much time trying to achieve a “great” design. Early on in a project this is rather futile as things will likely change as feedback comes in since it plays a key role in shaping your design. It’s a better use of time to get the design to a good spot, ship earlier, and iterate often.
```

Comment:
actually 1 more comment. i'm a little unclear on what `users are ultimately drawn back in the future` mean? maybe it's the combo of the words "back" and "future" used in the same sentence.

Reply:
👍  drawn back to the product because its ease of use. Let me know if you think there is a better way to phrase this.

Reply:
ah okay! 🚢